### PR TITLE
link to switch to the old version of the app

### PR DESCRIPTION
**Pull request by enriquepablo**
link to switch to the old version of the app.

Check the `react/i18n/l10n/en.json` and `react/i18n/l10n/sv.json` diffs for the text of the link and its tooltip in both english and svenska.

### CODE REVIEW

File: react/i18n/l10n/en.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
   "emails.invalid_email": "The entered value does not look like an email",
   "emails.long_description": "You can connect one or more email addresses with your eduID \n          account and select one to be your primary email address.",
   "emails.main_title": "Email addresses",
+  "emails.missing": "You must provide an emai address",
```
Comment (fredrikt) [inline react/i18n/l10n/en.json]: emai -> email

File: react/i18n/l10n/en.json [hunk 2]
```diff
@@ -117,6 +120,7 @@
   "nins.unconfirmed_nin": "Unconfirmed NIN",
   "nins.valid_nin": "Valid NIN",
   "nins.verified": "Verified",
+  "nins.verified_no_rm": "You cannot remove your ve3rified National identity number",
```
Comment (fredrikt) [inline react/i18n/l10n/en.json]: ve3rified -> verified

File: react/i18n/l10n/en.json [hunk 3]
```diff
@@ -58,6 +59,8 @@
   "error_lookup_mobile_task": "Problem looking up the phone number",
   "error_navet_task": "Communication problem with Navet",
   "faq_link": "For more information see the",
+  "foot.change-version": "Use old version of the app",
+  "foot.change-version-tip": "This is an experimental version. If you meet any problem while using the app, you can switch to the old version.",
```
Comment (fredrikt) [inline react/i18n/l10n/en.json]: meet -> experience

File: react/i18n/l10n/sv.json [hunk 4]
```diff
@@ -58,6 +59,8 @@
   "error_lookup_mobile_task": "Problem looking up the phone number",
   "error_navet_task": "Communication problem with Navet",
   "faq_link": "För mer information, se ",
+  "foot.change-version": "Use old version of the app",
+  "foot.change-version-tip": "This is an experimental version. If you meet any problem while using the app, you can switch to the old version.",
```
Comment (fredrikt) [inline react/i18n/l10n/sv.json]: meet -> experience

File: react/i18n/l10n/sv.json [hunk 5]
```diff
@@ -117,6 +120,7 @@
   "nins.unconfirmed_nin": "Obekräftad personnummer",
   "nins.valid_nin": "Valid NIN",
   "nins.verified": "Bekräftat",
+  "nins.verified_no_rm": "You cannot remove your ve3rified National identity number",
```
Comment (fredrikt) [inline react/i18n/l10n/sv.json]: ve3rified -> verified

File: react/i18n/src/i18n-messages.json [hunk 6]
```diff
@@ -759,6 +759,14 @@
     "id": "header.logout",
     "defaultMessage": "Logout"
   },
+  {
+    "id": "foot.change-version",
+    "defaultMessage": "Use old version of the app"
+  },
+  {
+    "id": "foot.change-version-tip",
+    "defaultMessage": "This is an experimental version. If you meet any problem while using the app, you can switch to the old version."
```
Comment (fredrikt) [inline react/i18n/src/i18n-messages.json]: meet -> experience

File: react/src/i18n-messages.js [hunk 7]
```diff
@@ -1115,9 +1115,20 @@ const msgs = {
...
 
+   'foot.change-version': (
+        <FormattedMessage
+          id="foot.change-version"
+          defaultMessage={`Use old version of the app`} />),
+
+   'foot.change-version-tip': (
+        <FormattedMessage
+          id="foot.change-version-tip"
+          defaultMessage={`This is an experimental version. If you meet any problem while using the app, you can switch to the old version.`} />),
```
Comment (fredrikt) [inline react/src/i18n-messages.js]: meet -> experience

File: react/i18n/l10n/sv.json (hunk 4 above)
Comment (enriquepablo) [reply react/i18n/l10n/sv.json]: In this and the next I disagree - it should be swedish ;-)

But of course I'll correct them for the benefit of the translator :)

**Pull request merged**